pstress/pstress-run.sh

```diff
@@ -1326,22 +1167,7 @@ EOF
       CRASH_CHECK=0
     else
       if [ ${SAVE_SQL} -eq 1 ]; then
-        if [ "${VALGRIND_RUN}" == "1" ]; then
-          if [ ${VALGRIND_ERRORS_FOUND} -ne 1 ]; then
-            echoit "Not saving anything for this trial (as SAVE_TRIALS_WITH_CORE_OR_VALGRIND_ONLY=1, and no issue was seen), except the SQL trace (as SAVE_SQL=1)"
-          fi
-        else
-          echoit "Not saving anything for this trial (as SAVE_TRIALS_WITH_CORE_OR_VALGRIND_ONLY=1, and no issue was seen), except the SQL trace (as SAVE_SQL=1)"
```

Comment:
Why else case is also removed with valgrind code. Please fix.

Reply:
Fixed